content/deep-dive/monitoring/_index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Monitoring
+weight: 9
+---
+
+Monitoring is critical for large distributed systems like TiKV. It alarms ops
```

Comment:
```suggestion
Monitoring is critical for large distributed systems like TiKV. It can alarm operators
```

Can we turn this into a list? "alarm operators when there is an abnormality, provide at-a-glance system health, and assist in performing diagnostics."